apps/modernization-ui/src/apps/page-builder/constant/constant.ts

```diff
@@ -276,6 +282,36 @@ const arithOperator = [
     }
 ];
 
+const nonDateCompare = [
+    {
+        name: 'Equal to',
+        value: '='
+    },
+    {
+        name: 'No equal to',
+        value: '!='
+    }
+];
```

Comment:
Question: Would it be better to have these constants defined in a file within the package of the component that uses them (such as the `BusinessRuleLibrary` folder)? This will prevent us having one giant `constant` file.